test/unit/steps/appointee/appointee-name/Appointee-name.test.js

```diff
@@ -79,29 +79,58 @@ describe('AppointeeName.js', () => {
     });
   });
 
-  [
-    'Mr,HARRY,POTTER',
-    'Mr,harry,potter',
-    'Mr,haRRy,pOttEr',
-    'Mr,harry John,pOttEr'
-  ].forEach(item => {
-    describe(`answers() appellant full name # ${item}`, () => {
+const Name = 'Mr,HARRY,POTTER-Smith';
```

Comment:
variable should either be all CAPS because it's constant or lower case. By convention variable names are lowercase, class names are capitalised